src/aiperf/common/tokenizer_display.py

```
@@ -261,13 +293,29 @@ def display_tokenizer_ambiguous_name(
     _display_panel("Ambiguous Tokenizer Name", content, console)
 
 
+def _reproduce_traceback(name: str) -> str | None:
+    """Try loading the tokenizer to capture the full traceback for diagnostics."""
+    import traceback
+
+    try:
+        from transformers import AutoTokenizer
+
+        AutoTokenizer.from_pretrained(name, trust_remote_code=True)
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)